crates/bevy_window/src/window.rs

```diff
@@ -37,6 +37,7 @@ pub struct Window {
     id: WindowId,
     width: u32,
     height: u32,
+    maximized: bool,
```

Comment:
This state will be hard to keep "up to date" because users can resize the window. Additionally, maximizing isn't supported on some platforms (android, ios, web), but this would happily report it as maximized. I think (for now) we should probably just drop it to avoid mis-reporting the state.